archive/tar_test.go

```diff
@@ -182,10 +180,14 @@ func testTar(t *testing.T, when spec.G, it spec.S) {
 				absoluteFilePath, err := filepath.Abs(filepath.Join("testdata", "dir-to-tar"))
 				h.AssertNil(t, err)
 
-				h.AssertNil(t, archive.WriteTarArchive(file, absoluteFilePath, uid, gid))
+				file, err := os.Create(filepath.Join(tmpDir, "tar_test-go.tar"))
+				h.AssertNil(t, err)
+				defer file.Close()
```

Comment:
closing the same file multiple times again

Reply:
See prior comment. We can continue the conversation there.